File: mvvm/src/main/java/com/loror/mvvm/net/SimpleProgressListener.java
package com.loror.mvvm.net;

import com.loror.lororUtil.http.ProgressListener;

public abstract class SimpleProgressListener implements ProgressListener {

    public void finish(boolean success) {
        if (success) {
            this.transing(-1.0F, 0, 0L);
        } else {
            this.transing(-2.0F, 0, 0L);
        }
    }
}

File: mvvm/src/main/java/com/loror/mvvm/net/SimpleProgressListener.kt
package com.loror.mvvm.net

import com.loror.lororUtil.http.ProgressListener

abstract class SimpleProgressListener : ProgressListener {

    override fun finish(success: Boolean) {
        if (success) {
            transing(-1f, 0, 0)
        } else {
            transing(-2f, 0, 0)
        }
    }
}